### ISSUE TITLE
oncePerFixture function does not work as expected

### ISSUE THREAD

**Issue by djiest** [DevExpress/testcafe#6453]
### What is your Test Scenario?
I would like to use the `oncePerFixture` functionality from the [`testcafe-once-hook` module](https://github.com/DevExpress/testcafe-once-hook)
 
### What is the Current behavior?
With the following setup, the url "'https://myapp.local/setup?foo=bar'" is never navigated to.

```
import { oncePerFixture } from 'testcafe-once-hook';

const setupDb = oncePerFixture(async t => {
    await t.navigateTo('https://myapp.local/setup?foo=bar');
});


fixture `Setup db`
    .beforeEach(setupDb);

    test('Test1', async t => {
        return true;
    });

```
 
### What is the Expected behavior?
I expect that before the tests once per fixture the url "'https://myapp.local/setup?foo=bar'" is called.
 
### What is your web application and your TestCafe test code?
Running above code will demonstrate that the url is never called.
The module used is https://github.com/DevExpress/testcafe-once-hook

### Steps to Reproduce:
Run above example with an existing URL by choice.
 
### Your Environment details:
 
* testcafe version:                   1.15.1
* node.js version:                    14.16.0
* command-line arguments:   testcafe "firefox" .\test.js
* browser name and version:  Firefox 90
* platform and version:           Windows
* other:                                   also see https://stackoverflow.com/questions/68589012/call-external-url-to-setup-and-teardown-in-testcafe

**Comment by AlexKamaev** [comment]
I was able to reproduce the issue.

**Comment by djiest** [comment]
I have done a rollback of Testcafe to version 1.14.2 and with that version I do get the expected result, namely navigating to `https://myapp.local/setup?foo=bar`

**Comment by Aleksey28** [comment]
Fixed in the `testcafe-once-hook@0.0.4`
